lua/lspconfig/server_configurations/qmlls.lua

```diff
@@ -2,7 +2,7 @@ local util = require 'lspconfig.util'
 
 return {
   default_config = {
-    cmd = { 'qmlls' },
+    cmd = { '/usr/bin/qmlls6' },
```

Comment:
this is not correct you can't hardcode this path.

Reply:
Under arch, /usr/bin/qmlls6 exists

Reply:
I don't think other people store this executable in this path.  and you break it on windows. so you should config it in your local i think.

Reply:
You are right, it should be replaced by qmlls6

Reply:
checked it Cmakelist 6 is qt quick version . it looks like generate executable with a version bump. so can we auto generate this executable name ? then it can adapt the version 7 8 9 etc